src/System.Security.Permissions/ref/System.Security.Permissions.csproj

```diff
@@ -4,16 +4,24 @@
   <PropertyGroup>
     <ProjectGuid>{07CAF142-B259-418E-86EF-C4BD8B50253E}</ProjectGuid>
     <!-- UAPvNext is not yet mapped to netstandard2.0, manually duplicate this ref -->
-    <PackageTargetFramework>netstandard2.0;$(UAPvNextTFM)</PackageTargetFramework>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard'">netstandard2.0;$(UAPvNextTFM)</PackageTargetFramework>
```

Comment:
We shouldn't even need this any longer as UAP supports netstandard2.0. IIRC we removed most of these from the repo already.